package.json

```diff
@@ -162,6 +162,7 @@
     "qs": "^6.5.0",
     "rc-slider": "^8.6.2",
     "react-css-transition-replace": "^3.0.2",
+    "react-head": "^3.0.0",
```

Comment:
@zephraph - `react-head` looks good, but wondering if you investigated any other possibilities? Did this one seem most useful long-term?

Reply:
I did. `react-helmet` is the most popular option. While it gives a lot of flexibility, it's pretty heavy and it's _not_ async friendly. There's `react-helmet-async` which is a variation of `react-helmet` but it doesn't use `react-side-effects`. It's also still listed as unstable and there are outstanding issues that causes it to be a problem. `react-head` uses a bit of a simpler approach then the others (mainly relying on portals). It's written by one of the tech leads at the NYT, it's well tested, and best of all it's very small.

Reply:
Beautiful, ty 👍